client/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  preset: '@vue/cli-plugin-unit-jest/presets/typescript-and-babel'
+}
```

Comment:
Again, very different from ours (https://github.com/bcgov/lear/blob/master/coops-ui/jest.config.js). I'm thinking either you have too little or we have too much. I haven't experimented with these settings very much.

Reply:
well the repo's were created at different times.  I presume it could just be updates to the various packages changing their interfaces and options and the vue-cli simply keeping up with the newest changes.  I haven't written or attempted to run any jest tests yet tho, I will certainly look to your settings if  things are not working.